craft_application/models/constraints.py

```diff
@@ -180,12 +179,12 @@ def validate(value: str) -> str:
         strict=False,
         coerce_numbers_to_str=True,
         title="version string",
-        description="A string containing the version of the project",
+        description="The version of the project, enclosed in quotation marks.",
         examples=[
-            "0.1",
-            "1.0.0",
-            "v1.0.0",
-            "24.04",
+            '"0.1"',
+            '"1.0.0"',
+            '"v1.0.0"',
```

Comment:
These two can be things like `version: v1.0.0` and YAML will correctly handle it.

Reply:
I'm fine with documenting something stricter as long as we're aware it's not *technically* what we require.